### ISSUE TITLE
Hiveos ETC/Zil + Kaspa Not working

### ISSUE THREAD

**Issue by cuneytavci** [develsoftware/GMinerRelease#825]
Hi I m trying to setup flightsheet but Gminer 3.09 just mining Etc/zil only. Maybe this is Hiveos problem but I wanted to inform

**Comment by Jorgelote** [comment]
yes, it is working you just need to put it manually like: "--dalgo kheavyhash --dserver pool.br.woolypooly.com:3112 --duser x  " (there would be the kaspa wallet) and then, it might look that it is not working, but look at the hive shell and it is mining kaspa this way, just that hiveos does not display it on the webpage

**Comment by Jorgelote** [comment]
just be sure no not enter the " "

**Comment by cuneytavci** [comment]
> yes, it is working you just need to put it manually like: "--dalgo kheavyhash --dserver pool.br.woolypooly.com:3112 --duser x " (there would be the kaspa wallet) and then, it might look that it is not working, but look at the hive shell and it is mining kaspa this way, just that hiveos does not display it on the webpage

thx man its working.. I think hiveos must fix something to work their flightsheet assistant work
